### ISSUE TITLE
A11y: tabbed interface/router

### ISSUE THREAD

**Issue by patrickhlauke** [bitwarden/browser#1982]
## Describe the Bug

Currently, the tabbed interface ("Tab", "My vault", "Send", "Generator", "Settings") is marked up and exposed as a series of links. The currently active tab is not identified explicitly as such for assistive technology users.

Functionally, these should either be exposed as buttons (and the currently active one denoted programmatically), or - taking this further - the whole tabbed interface should be implemented using the appropriate ARIA pattern https://www.w3.org/TR/wai-aria-practices-1.2/#tabpanel (for instance, https://www.w3.org/TR/wai-aria-practices-1.2/examples/tabs/tabs-1/tabs.html)

## Steps To Reproduce

1. using a screen reader, open the extension popup
2. navigate through the tabbed interface controls

## Expected Result

* the tab controls are either exposed as buttons (and the currently active one is explicitly denoted as such), or
* the tabbed interface is exposed as a "proper" ARIA tabbed interface

## Actual Result

* the tab controls are exposed/announced as links
* the currently active control is denoted only visually - no announcement is made by assistive technologies that identifies the currently active one

## Screenshots or Videos

![Tabbed interface](https://user-images.githubusercontent.com/895831/127712926-8900f580-4562-4f76-bbcb-c4a2722dcb56.png)

## Environment

- Operating system: Windows 10
- Browser: Chrome 92
- Build Version (go to "Settings" → "About" in the app): 1.51.1

## Additional Context

If going for the simpler option of using `<button>`, denote the currently active one with `aria-pressed="true"`, and the inactive ones with `aria-pressed="false"`

**Comment by patrickhlauke** [comment]
An alternative to just programmatically denote the currently active control (which is visually highlighted in blue, compared to the grey of the other controls) would be to at the very least add `aria-current="location"` to the highlighted one (see https://tink.uk/using-the-aria-current-attribute/)

**Comment by patrickhlauke** [comment]
This issue/bug is still present (and I may be able to tackle this in a PR)

**Comment by dwbit** [comment]
Thanks @patrickhlauke, I've removed the label.

**Comment by patrickhlauke** [comment]
Made a PR for this https://github.com/bitwarden/browser/pull/2505
